lib/modules/datasource/github-release-attachments/index.ts

```diff
@@ -55,7 +55,7 @@ export class GithubReleaseAttachmentsDatasource extends Datasource {
     ttlMinutes: 1440,
     namespace: `datasource-${GithubReleaseAttachmentsDatasource.id}`,
     key: (release: GithubRestRelease, digest: string) =>
-      `${release.html_url}:${digest}`,
+      `findDigestFile:${release.html_url}:${digest}`,
```

Comment:
do we know the function name inside the decorator? so we can automatically prefix the key?

Reply:
Yes, later I plan to do exactly that